packages/international-types/index.ts

```diff
@@ -1,5 +1,6 @@
 export type LocaleValue = string | number | boolean | null | undefined | Date;
 export type BaseLocale = Record<string, LocaleValue>;
+export type BaseLocales = Record<string, any>;
```

Comment:
Could we restrict more this type? We only want to accept objects with `() => import(...)` statements:

```suggestion
export type BaseLocales = Record<string, () => Promise<{ default: any }>>;
```